webapp/templates/jury/contests.html.twig

```diff
@@ -84,12 +84,15 @@
 
     <h3>All available contests</h3>
 
-    {{ macros.table(contests_table, table_fields, num_actions) }}
+    {{ macros.table(contests_table, table_fields, num_actions, null, "jury_contests_delete") }}
 
     {% if is_granted('ROLE_ADMIN') %}
         <p>
             {{ button(path('jury_contest_add'), 'Add new contest', 'primary', 'plus') }}
             {{ button(path('jury_import_export', {'_fragment':'contests'}), 'Import contest', 'primary', 'upload') }}
+            <button type="submit" form="multidelete" class="btn btn-danger" name="multidelete_submit">
```

Comment:
Should we add some javascript to disable this button if nothing is selected? We can probably do that in the macro?